src/main/java/javaserver/App.java

```diff
@@ -1,31 +1,47 @@
 package javaserver;
 
+import java.io.File;
 import java.io.IOException;
 import java.net.ServerSocket;
 
 public class App {
 
-	protected static final int PORT = 5000;
+	public static String name = "Cat's Java Server";
+	protected static final int DEFAULT_PORT = 5000;
+	protected static int port;
+	protected static final String DEFAULT_PUBLIC_DIRECTORY = "public";
+	protected static PublicDirectory publicDirectory = new PublicDirectory(DEFAULT_PUBLIC_DIRECTORY);
 	protected static Server server;
-	protected static boolean isOn = false;
 
 	public static void main(String[] args) throws IOException {
-		setUpServer();
+		setUpServer(args);
 		runServer(server);
 	}
 
-	protected static void setUpServer() throws IOException {
-		server = new Server(new ServerSocket(PORT), new ThreadManager());
+	protected static void setUpServer(String[] args) throws IOException {
+		port = ArgHandler.getPort(args, DEFAULT_PORT);
+		setDirectory(args);
```

Comment:
What about `publicDirectory = initializeDirectoryRouter();` instead? Just to make it more explicit as to what is going on?